packages/server/lib/files.coffee

```diff
@@ -13,7 +14,7 @@ module.exports = {
     readFn(filePath, options.encoding or "utf8")
     .then (contents) ->
       {
-        contents: contents
+        contents: if path.extname(filePath) is ".yml" or path.extname(filePath) is ".yaml" then yaml.safeLoad(contents) else contents
```

Comment:
Ok, I've pulled out the if-else inline statement. It's more wordy. I'm tempted to make yaml.safeLoad to be async so it can be included in readFn (line 9).